src/main/java/com/team4/testingsystem/entities/ContentFile.java

```diff
@@ -20,33 +22,34 @@
     @Column(name = "id")
     private Long id;
 
-    @OneToOne
-    @JsonIgnore
-    @JoinColumn(name = "question_id", nullable = false, referencedColumnName = "id")
-    private Question question;
+    @ManyToMany(cascade = CascadeType.ALL)
+    @JoinTable(
+            name = "question_content_file",
+            joinColumns = @JoinColumn(name = "content_file_id"),
+            inverseJoinColumns = @JoinColumn(name = "question_id")
+    )
+    List<Question> questions = new ArrayList<>();
```

Comment:
Why do we need to create empty collections here?

Reply:
This is to avoid getting a NullPointerException.